src/core/tests/type_prop/bitwise_ops.hpp

```diff
@@ -0,0 +1,1020 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/core/dimension_tracker.hpp"
+#include "openvino/op/util/attr_types.hpp"
+
+using namespace ov;
+using namespace testing;
```

Comment:
Try to remove it or be consistent in using the namespace as in code is often used `ov::....`

Consider remove it and add required using like
`using ov::op::v0::Parameter` then in test use just `Parameter` etc.

Reply:
removed all ov:... from namespace, modified to use just Parameter